Sources/SnapshotTesting/Strategy/CGSize.swift

```diff
@@ -0,0 +1,88 @@
+#if os(iOS) || os(macOS) || os(tvOS)
+import CoreGraphics
+#endif
+
+public enum Orientation {
+  case horizontal
+  case vertical
+}
+
+extension CGSize {
+  static func iPhoneSe(_ orientation: Orientation) -> CGSize {
```

Comment:
Minor thing: do we wanna do `iPhoneSE` instead? I'm keen on sticking with our camelcasing guns.

Reply:
(I could remove this file if we can't figure it out and the other changes should hopefully still be useful!)